TWLight/users/models.py

```diff
@@ -422,7 +422,7 @@ class Meta:
         app_label = "users"
         verbose_name = "authorization"
         verbose_name_plural = "authorizations"
-        unique_together = ("user", "partner", "stream", "date_authorized")
```

Comment:
Yep, this was one of the earliest bits of logic for authorizations, and our use-case for them has dramatically firmed up since then.